app/models/enterprise.rb

```diff
@@ -479,7 +477,7 @@ def image_url_for(image, name)
     return unless image.variable?
 
     image_variant_url_for(image.variant(name))
-  rescue ActiveStorage::Error, MiniMagick::Error, ActionView::Template::Error => e
+  rescue StandardError => e
```

Comment:
I think it's worth a comment explaining why, eg that it gracefully catches MiniMagic errors without the gem being loaded.

Reply:
I don't think that we need to mention Minimagick here because it doesn't matter anymore. We don't care what error is raised. Maybe the commit message should have explained that better...